src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkImageBrowser.tsx

```diff
@@ -17,16 +17,17 @@ import {
 
 interface ArtworkBrowserProps {
   images: ArtworkImageBrowser_artwork["images"]
+  sliderRef?(slider: Slider): void
 }
 
-export const ArtworkImageBrowser: React.SFC<ArtworkBrowserProps> = props => {
+export const ArtworkImageBrowser = (props: ArtworkBrowserProps) => {
   return (
     <>
       <Media at="xs">
-        <SmallArtworkImageBrowser images={props.images} />
+        <SmallArtworkImageBrowser {...props} />
       </Media>
       <Media greaterThan="xs">
-        <LargeArtworkImageBrowser images={props.images} />
+        <LargeArtworkImageBrowser {...props} />
```

Comment:
Why use the spread here?

Reply:
These two components take the same props as the wrapper component, so why not pass them all along.